src/blocks/price.css

```diff
@@ -0,0 +1,18 @@
+/* Price */
```

Comment:
it's a part of `product-card` block, move it and all `product-card's` 
elements into its file